### ISSUE TITLE
Error updating $schema if editor no longer has focus

### ISSUE THREAD

**Issue by vvcomsys** [microsoft/vscode-azurearmtools#389]
Repro steps:

1.
2.

Action: queryUpdateSchema
Error type: Error
Error Message: TextEditor#edit not possible on closed editors

Version: 0.8.2
OS: win32
Product: Visual Studio Code
Product Version: 1.40.0
Language: en

<details>
<summary>Call Stack</summary>

```
d.edit extensionHostProcess.js:493:41
AzureRMTools.<anonymous> extension.bundle.js:127:181867
extension.bundle.js:127:174871extension.bundle.js:127:174871
n extension.bundle.js:127:174619
AzureRMTools.replaceSchema extension.bundle.js:127:181575
AzureRMTools.<anonymous> extension.bundle.js:127:181275
o extension.bundle.js:127:174675
```

</details>

**Comment by StephenWeatherford** [comment]
@vvcomsys Interesting.  I'm guessing you closed the editor before clicking to update the schema?  Or did something else happen?  Thanks.

Also, I assume it works if you don't close the editor first?

**Comment by StephenWeatherford** [comment]
Okay, I see that even just tabbing away from the editor can do this.

**Comment by StephenWeatherford** [comment]
Fixed in #389, not yet released. Thanks for reporting. In the meantime, it should work if you don't close or change tabs before you click to update the schema. Thanks!
